### PR TITLE
Fix the copy css function and some other changes

### PR DESCRIPTION
**Pull request by Moshem123**
- Fix copy css function
- Add prevState logic where it was needed
- Remove unused variable
- Add .editorconfig

### CODE REVIEW

File: src/NewWindow.js [hunk 1]
```diff
@@ -153,12 +165,28 @@ function copyStyles(source, target) {
...
+    }
+    if (rules) {
       const newStyleEl = source.createElement('style')
 
       // Write the text of each rule into the body of the style element
       Array.from(styleSheet.cssRules).forEach(cssRule => {
-        newStyleEl.appendChild(source.createTextNode(cssRule.cssText))
+        const { cssText, type } = cssRule;
+        let returnText = cssText;
+        if ([3, 5].includes(type)) {
```
Comment (rmariuzzo) [inline src/NewWindow.js]: @Moshem123 can we make it clear what does `3` and `5` means? Probably, declaring them as constant and giving a self explaining name. 🙇

**Comment by Moshem123** [comment]
@rmariuzzo done.
Please confirm :)

File: src/NewWindow.js [hunk 2]
```diff
@@ -22,7 +23,7 @@ class NewWindow extends React.PureComponent {
     url: '',
     name: '',
     title: '',
-    features: {},
+    features: { width: '600px', height: '640px' },
```
Comment (Moshem123) [inline src/NewWindow.js]: If there's no default width and height, the window won't open

**Comment by rmariuzzo** [comment]
Closes: https://github.com/rmariuzzo/react-new-window/issues/3

**Pull request merged**